standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -1400,6 +1405,43 @@ public void create_database(final Database db)
     }
   }
 
+  @Override
+  public void create_database_req(final CreateDatabaseRequest req)
+          throws AlreadyExistsException, InvalidObjectException, MetaException {
+    Database db = req.getDatabase();
+    boolean skipFSWrites = req.isSkipFSWrites();
```

Comment:
Nit: We directly pass this value in L#1427 instead of declaring a variable here.

Reply:
Sure.